### ISSUE TITLE
Error for field empty - index parameter is out of range

### ISSUE THREAD

**Issue by davideciarmiello** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#579]
Hi. i'm mapping a table that i can't edit, and the table have a field with all whitespaces : '   '.
col.NameHumanCase is '', and i receive a error for this line:

if (char.IsDigit(col.NameHumanCase[0]))
                col.NameHumanCase = "_" + col.NameHumanCase;

Can you solve the problem? THanks :)

**Comment by sjh37** [comment]
Hi. Search for `col.NameHumanCase = CleanUp(col.DbName);` and add a line below it.
```c#
if (string.IsNullOrWhiteSpace(col.NameHumanCase))
{
    col.NameHumanCase = "Unknown";
    col.Hidden = true; // optional
}
```

Marking a column as Hidden means the column will not be output in the POCO class. If you wish to edit the columns data, then give `col.NameHumanCase` a suitable name, and remove `col.Hidden=true;`

See if that helps

**Comment by davideciarmiello** [comment]
Hi. Yes the solution resolve the problem.

I wanted to make the post only to report the problem and to have solved it in the next version.

Your project is greatful! Thanks!

**Comment by sjh37** [comment]
Now released as part of v3.2.0
